cartridges/int_bolt_embedded_sfra/cartridge/controllers/Bolt.js

```diff
@@ -148,4 +149,59 @@ server.post('GetAccount', function (req, res, next) {
     return next();
 });
 
+server.post('CreateCompleteAccount', function (req, res, next) {
+    try {
+        if (!httpUtils.getAuthenticationStatus()) {
+            return httpUtils.errorResponse('Request is not authenticated.', 401, res, next);
+        }
+
+        var httpParameterMap = request.getHttpParameterMap();
+        var requestBodyString = httpParameterMap.get('requestBodyAsString') ? httpParameterMap.requestBodyAsString : null;
+        var requestBody = JSON.parse(requestBodyString);
+        var boltProfile = requestBody.profile;
+        var boltAddresses = requestBody.addresses;
+        var boltPayments = requestBody.payment_methods;
+
+        if (!boltProfile) {
+            return httpUtils.errorResponse('Missing profile in the request body.', 400, res, next);
+        }
+        if (!boltProfile.email) {
+            return httpUtils.errorResponse('Missing email in the request body profile.', 400, res, next);
+        }
+
+        var platformAccountID = UUIDUtils.createUUID();
```

Comment:
Looks like every time it is called, it will create a new SFCC customer. shouldn't this platform account ID be passed from Bolt?

Reply:
yes, talked with Paul offline, he will pass the platform account ID, I will make a separate PR.